src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.cpp

```diff
@@ -1434,8 +1460,14 @@ void C2_MacroAssembler::string_compare(Register str1, Register str2,
   // load first parts of strings and finish initialization while loading
   {
     if (str1_isL == str2_isL) { // LL or UU
-      // check if str1 and str2 is same pointer
-      beq(str1, str2, DONE);
+#ifdef ASSERT
```

Comment:
Can we add the some comment about 8-bytes alignment below at line 1520?
I know it's redundant, but as the code is getting more complicated, it might be good to add it to improve the readability.

Reply:
Sure. Done.